### ISSUE TITLE
file_renamer fails to parse chapter_number correctly when there is a whitespace in chapter_title

### ISSUE THREAD

**Issue by Jk8c19** [Inpacchi/Manga-Tagger#15]
**Manga Tagger Version:** v1.0.3 [973fd1e](https://github.com/Inpacchi/Manga-Tagger/commit/973fd1ed445f13a4d5fae4f0c559aafbdf0f3ec7)
**Description of the issue:** file_renamer fails to parse chapter_number correctly when there is a whitespace in chapter_title, I've come across a handful of series where the name looks something like "ch. 001".

**Detailed findings:** found that the culprit was inside the while loop that creates the chapter_number variable, if it encounters a whitespace it breaks out and you end up with "000" as your parsed chapter_number.

My dirty workaround was to add an else if to skip over these adding to the index to prevent it from breaking out:

```python
    while i < len(chapter_title):
        substring = chapter_title[i]
        LOG.debug(f'substring: {substring}')

        if substring.isdigit() or substring == '.':
            chapter_number += chapter_title[i]
            i += 1

            LOG.debug(f'chapter_number: {chapter_number}')
            LOG.debug(f'Iterator i: {i}')
        elif substring == ' ':
            i += 1

            LOG.debug(f'Skipping whitespace character')
            LOG.debug(f'Iterator i: {i}')
        else:
            break
```
Unsure if this covers all eventualities but at least it no longer cares if there are spaces between the delimiter and chapter number.

**Comment by Inpacchi** [comment]
Fixed this in 1306de41d0c623d97c616918af0abdfd87eb4d74 and 6a9851ddd4e686bf0ae6104252de83e3fd606c5b. I originally fixed this a few weeks ago by completely stripping whitespace, but as I just looked at it, I saw how it might pose a problem. The next fix I did creates a separate variable to strip whitespace from.

**Comment by Inpacchi** [comment]
@Jk8c19 I reverted 6a9851ddd4e686bf0ae6104252de83e3fd606c5b. I forgot I had tests set up and when I just ran them, 4 tests failed due to this change. Guess I should set up my CI pipeline now, hah...
